nextjs-app/components/common/CountdownTimer.tsx

```diff
@@ -0,0 +1,100 @@
+"use client";
+
+import { FC, useEffect, useRef, useState } from "react";
+import Image from "next/image";
+
+const DEADLINE = new Date("Feb 28, 2025 18:00:00").getTime();
+
+const TimeBlock: FC<{ time: string; unit: string }> = ({ time, unit }) => {
+  return (
+    <>
+      <div className="relative p-3 size-[72px] text-h1 text-white">
+        <Image
+          src="/images/index-banner/time-block.png"
+          alt="time-block"
+          fill
+          className="absolute w-full h-full object-cover -z-10"
+        />
+        {time}
+      </div>
+      <div className="text-blue-8 text-subtitle-md">{unit}</div>
+    </>
+  );
+};
+
+const CountdownTimer = () => {
+  const [time, setTime] = useState<{
+    days: string;
+    hours: string;
+    minutes: string;
+  }>({
+    days: "00",
+    hours: "00",
+    minutes: "00",
+  });
+
+  const intervalRef = useRef<NodeJS.Timeout | null>(null);
```

Comment:
```suggestion
  const intervalRef = useRef<ReturnType<typeof setInterval>t | null>(null);
```

Since this is a client component, I believe using this type will be more suitable